src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -113,6 +111,15 @@ public bool TryProcessCommand(IExecutionContext context, string input, Container
                         context.Output(input);
                         context.Debug("Paused processing commands until '##[{actionCommand.Data}]' is received");
                         _stopToken = actionCommand.Data;
+                        if (_registeredCommands.Contains(actionCommand.Data) || string.IsNullOrEmpty(actionCommand.Data))
+                        {
+                            var telemetry = new JobTelemetry
+                            {
+                                Message = $"Invoked ::stopCommand:: with token: [{actionCommand.Data}]",
```

Comment:
let's run through secret masker for this, since the `actionCommand.Data` might be a GITHUB_TOKEN.